ocs_ci/framework/pytest_customization/reports.py

```diff
@@ -43,11 +43,21 @@ def pytest_runtest_makereport(item, call):
             log_file = logging.getLogger().handlers[1].baseFilename
         extra.append(pytest_html.extras.url(log_file, name='Log File'))
         report.extra = extra
+        item.session.results[item] = report
+    if report.skipped:
+        item.session.results[item] = report
+
+
+def pytest_sessionstart(session):
+    """
+    Prepare results dict
+    """
+    session.results = dict()
 
 
 def pytest_sessionfinish(session, exitstatus):
     """
     send email report
     """
     if ocsci_config.RUN['cli_params'].get('email'):
-        email_reports()
+        email_reports(session)
```

Comment:
how does this work with multiple pipelines?

Reply:
@vasukulkarni what do you men by _multiple pipelines_?